src/map/lua/luautils.cpp

```diff
@@ -4128,6 +4128,8 @@ namespace luautils
     {
         TracyZoneScoped;
 
+        ShowDebug("LoadEventScript: Checking Function %s\n", functionName);
```

Comment:
I'm in favour of these messages, but perhaps not for Debug. I've been considering if we need `ShowDev` for REALLY extended logging...

Reply:
Accidentally check it back in after it was removed.  Removed again to match the current base.